modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -778,6 +778,13 @@ private IgniteInternalFuture<SnapshotOperationResponse> initLocalSnapshotStartSt
         if (!CU.baselineNode(cctx.localNode(), cctx.kernalContext().state().clusterState()))
             return new GridFinishedFuture<>();
 
+        if (log.isInfoEnabled()) {
+            log.info("Starting local snapshot operation [snpName=" + req.snapshotName() +
```

Comment:
let's log full SnapshotOperationRequest (use GridExcludeString for useless fields)